Homework6/Task1Logger/Program.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Reflection;
+using AttributeLibrary;
+
+namespace Task1Logger
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Logger TestLogger = new("testing.json");
```

Comment:
make all the variables lowercase. 
https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions
https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md